src/goose/toolkit/prompts/memory.jinja

```diff
@@ -0,0 +1,30 @@
+You have access to a memory system that can store and recall important information across conversations.
```

Comment:
I wonder if this will realise it means itself - may want to test this across a few models to check that out

Reply:
good point. Prompt engineering is not my greatest asset